config/database.yml

```diff
@@ -18,8 +18,8 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: db/test.sqlite3
+  database: test
 
 production:
   <<: *default
-  database: db/production.sqlite3
+  database: production
```

Comment:
To avoid collisions I usually use the following syntax for database names: `youth-match-v2_development` `youth-match-v2_production` etc.